zproject/prod_settings_template.py

```diff
@@ -244,6 +244,13 @@
         "attr_last_name": "last_name",
         "attr_username": "email",
         "attr_email": "email",
+
+        # The SAML attributes can be used to filter which organizations the user can access.
+        # "attr_org_membership" contains the attribute name which contains a list of
+        # organization subdomains the user is allowed to access. For the root subdomain,
+        # 'www' in the list will work, or any other of settings.ROOT_SUBDOMAIN_ALIASES.
+        # "attr_org_membership": "member",
```

Comment:
Let's move most of this documentation to `docs/production/authentication.md`; for this block in general we're only writing here what's essential to understand how to set the option.

Reply:
Done, wrote some documentation for `production/authentication-methods.md`